src/search/merge_and_shrink/merge_scoring_function_dfp.cc

```diff
@@ -130,10 +134,13 @@ static shared_ptr<MergeScoringFunction>_parse(options::OptionParser &parser) {
         "greedy=false),label_reduction=exact(before_shrinking=true,"
         "before_merging=false),max_states=50000,threshold_before_merge=1)\n}}}");
 
+    add_merge_scoring_function_options_to_parser(parser);
+
```

Comment:
Here you add a configurable verbosity (via the logger) to classes which did not previosuly have it. Just because it would be good here or is there a reason why this is needed in the current patch, e.g. you previosuly just passed Verbosity::Silent to a function, but now would need to create a new LogProxy object for each function call.

Reply:
from below this seems to be exactly the case. Can you still confirm, please.

Reply:
Yes, this is unrelated to this issue. I don't remember why I did this. Probably to have some kind of test case, I guess. I could also revert the changes done in M&S.

Reply:
As discussed, we decided to make the changes less invasive and replace the currently hard-coded silent verbosity of the MIASM scoring function to a hard-coded silent logger.